filebeat/input/winlog/input.go

```diff
@@ -117,14 +126,16 @@ runLoop:
 			continue
 		case !api.IsFile() && eventlog.IsChannelNotFound(openErr):
 			if !channelNotFoundErrDetected {
-				log.Errorw("Encountered channel not found error when opening Windows Event Log", "error", openErr)
+				log.Errorw(channelNotFoundError, "error", openErr)
 			} else {
-				log.Debugw("Encountered channel not found error when opening Windows Event Log", "error", openErr)
+				log.Debugw(channelNotFoundError, "error", openErr)
 			}
+			ctx.UpdateStatus(status.Degraded, fmt.Sprintf("%s: %v", channelNotFoundError, openErr))
```

Comment:
I think this should go after https://github.com/elastic/beats/pull/40163/files#diff-6a618961acc22d5887cb9fd1844390de34b37bcf90b6ec247799b94d047b5d90R129 since the first time is not considered an error and it is retried after https://github.com/elastic/beats/pull/40163/files#diff-6a618961acc22d5887cb9fd1844390de34b37bcf90b6ec247799b94d047b5d90R134